src/serializer/utils.js

```diff
@@ -81,3 +82,24 @@ export function getOrDefault<K, V>(map: Map<K, V>, key: K, defaultFn: () => V):
   invariant(value !== undefined);
   return value;
 }
+
+export function withDescriptorValue(
+  propertyNameOrSymbol: string | SymbolValue,
+  descriptor: void | Descriptor,
+  func: Function
```

Comment:
This function should be typed using generics and a specific function signature. All these functions passed here are inferred as `any` right now.

Reply:
This was done intentionally. Flow just gets confused and outputs about 20 errors if you try and make this something other than `any`. I spoke to the Flow team and they said that Flow struggles with this sort of pattern and its on their roadmap to fix. I'm not sure it's of any value to hack around Flow to make this work right now.